@xen-orchestra/web/src/components/alarms/DashboardAlarms.vue

```diff
@@ -64,7 +64,7 @@ const {
   wrapperProps,
 } = useVirtualList(
   computed(() => rawAlarms),
-  { itemHeight: () => (uiStore.isDesktopLarge ? 40 : 70) }
+  { itemHeight: () => (uiStore.isDesktopLarge ? 42 : 70) }
```

Comment:
```suggestion
  { itemHeight: () => (uiStore.isDesktopLarge ? 42 : 71) }
```

Got same issue in small screen